pymc/logprob/mixture.py

```diff
@@ -404,6 +418,8 @@ def logprob_MixtureRV(
 
     assert len(indices) > 0
 
+    # FIXME: This logic does not account for repeated indexes, in which case the Mixture values are not independent.
+    #  It over-counts the logp terms for values that were repeated during indexing
```

Comment:
Should this have a unit test when it's resolved?

Reply:
I don't know if we will ever resolve it. For now we don't allow creating such MixtureRVs in the first place. The ones that hit this branch don't do AdvancedIndexing